packages/app/src/app/overmind/namespaces/comments/actions.ts

```diff
@@ -523,22 +499,34 @@ export const onCommentAdded: Action<CommentAddedSubscription> = (
   { state },
   { commentAdded: comment }
 ) => {
-  state.comments.comments[comment.sandbox.id][comment.id] = comment;
+  // We only add it if it was NOT created by the user, cause then we
+  // have already set it, avoiding UI flickers
+  if (comment.user.id !== state.user?.id) {
```

Comment:
Maybe it would be better to do an `Object.assign` or a nullcheck. I can see that this breaks if someone has the sandbox open with two tabs, and the second tab doesn't receive the comment.

Reply:
I think this can be improved when we now define the ID in the client, will update from master and look at this after @SaraVieira polish is merged in